### DISCUSSION TITLE
Papermerge on FreeBSD 12.2-Release-p4

### DISCUSSION THREAD

**Discussion by a-w-a-t-8-7** [ciur/papermerge#348] [Q&A] [answered]
Hello, I am trying to install Papermerge on a FreeBSD system. I always fail at the same point

(.venv) [root@Test12 /PapermergeDMS]# ./manage.py check
System check identified some issues:

WARNINGS:
?: Papermerge can't find convert. Without it, image resizing is not possible.
        HINT: Either it's not in your PATH or it's not installed.
?: Papermerge can't find identify. Without it, it is not possible to count pages in TIFF.
        HINT: Either it's not in your PATH or it's not installed.
?: Papermerge can't find pdfinfo. Without it, Papermerge won't function properly. It won't be able to find out PDF files page count.
        HINT: Either it's not in your PATH or it's not installed.
?: Papermerge can't find pdftk. Without it, Papermerge won't be able to cut/paste PDF pages.
        HINT: Either it's not in your PATH or it's not installed.
?: Papermerge can't find pdftoppm. Without it, it not possible to extract images from PDF.
        HINT: Either it's not in your PATH or it's not installed.
?: Papermerge can't find tesseract. Without it, OCR of the documents is impossible.
        HINT: Either it's not in your PATH or it's not installed.

If i have a look to "/usr/local/bin" i can find "convert","identify","pdfinfo","pdftk","pdftoppm" and "tesseract". I have also checked if there are updates for the installed packages, but this is not the case.

Can anyone help me with this?

Regards

**Comment by ciur** [comment, marked as answer]
@a-w-a-t-8-7, you need to set [these configurations](https://papermerge.com/docs/Installation/settings.html#binary-dependencies) in your papermerge.conf.py file
